requirements.txt

```diff
@@ -0,0 +1 @@
+defusedxml==0.5.0
```

Comment:
Not sure if you want a requirements.txt, thoughts @alanjcastonguay?

Reply:
Sure.